src/Data/RdsData/Internal/Convert.hs

```diff
@@ -36,6 +42,16 @@ utcTimeToText :: UTCTime -> Text
 utcTimeToText = 
   T.pack . formatTime defaultTimeLocale "%Y-%m-%d %H:%M:%S"
 
+ulidToText :: ULID -> Text
+ulidToText = 
+  ULID.encode 26 . ULID.ulidToInteger
+
+textToUlid :: Text -> Either Text ULID
+textToUlid t = 
+  case ULID.decode 26 t of
+    (n, ""):_ -> ULID.ulidFromInteger n
```

Comment:
Shouldn't we be pattern matching on the singleton list? Although it's only possible for this function to return `[]`or `[val]`.

Reply:
I switched the code to use the `Read` instance with avoids having to deal with the list.